src/main/resources/assets/rockbottom/content/recipes/simple_furnace.json

```diff
@@ -1,6 +1,14 @@
 {
     "type": "construction_table",
     "knowledge": true,
+    "criteria": [
```

Comment:
did this not already have criteria? whoops

Reply:
Apparently not